src/firefly/js/ui/UploadTableChooser.js

```diff
@@ -82,23 +89,96 @@ function uploadSubmit(request, setUploadInfo)  {
     return false;
 }
 
-
 function existingTableSubmit(request,setUploadInfo) {
-    // todo: call this function for preloaded tables
-    const uploadInfo = {
-        serverFile: '${upload}//stuff/somename.tbl',
-        title: 'tbl title',
-        tbl_id: 'someTblId',
-        columns: [],
-        totalRows: 123,
-        tableSource: EXISTING_TBL_SOURCE,
-    };
-    setUploadInfo(uploadInfo);
-    dispatchHideDialog(dialogId);
+    if (!request) return false;
+    const tbl = getTblById('existing-table-list-ui');
+    const idx = tbl.highlightedRow;
+    const activeTblId = tbl.tableData.data[idx][3]; //tbl_id
+    const activeTbl = getTableUiByTblId(activeTblId);
+    const tableRequest = activeTbl.request;
+    const columnData = activeTbl.columns;
+    const columns = columnData.map((col) => col.visibility === 'show'? ({...col, use:true}) :  ({...col, use:false}));
+
+    const params = omitBy({
+        [ServerParams.COMMAND]: ServerParams.TABLE_SAVE,
+        [ServerParams.REQUEST]: JSON.stringify(tableRequest),
+        file_name: tableRequest.META_INFO.title,
+        file_format: Format.IPACTABLE,
+        save_to_temp: 'true'
+    }, isNil);
```

Comment:
i don't see why you are using `omitBy`. The only possible thing here is file_name, if it is require you should make sure it is defined.